webui/packages/eidolon-components/src/messages/chat-display-element.tsx

```diff
@@ -14,16 +14,16 @@ export interface ChatDisplayElementProps {
 
 export const ChatDisplayElement = ({machineUrl, rawElement, agentName, topLevel, userImage}: ChatDisplayElementProps) => {
   const getUserInput = (element: UserRequestElement) => {
-    // todo, make sure this renders file inputs correctly
-    let content = {...element.content}
+    let content = typeof element.content === "string" ? {body: element.content} : {...element.content}
     delete content["process_id"]
     if (Object.keys(content).length === 0) {
       return "*No Input*"
-    } else if (Object.keys(content).length === 1) {
-      content = content[Object.keys(content)[0]!]
+    } else if (Object.keys(content).length === 1 && Object.keys(content)[0] === "body") {
```

Comment:
IMO, we should format this case differently than a simple string input, but this is good and I'll change it later